src/modules/header/styles.css

```diff
@@ -0,0 +1,268 @@
+@custom-media --header-breakpoint (min-width: 768px);
+
+:root {
+  --header-menu-width: 12rem;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  line-height: 1; /* FIXME: in another place */
```

Comment:
This is something we should be probably getting rid off. I mean if I define a fixed height for the header the buttons are not stretched to the whole area.  And this height has to be defined because the main area should be moved down by that offset. (--app-bar-skip)

Reply:
What do you mean saying 'get rid off'?

Reply:
I mean if we have it there than buttons are not going to be stretched properly when you specify a height for app bar and imho it should by default

![image](https://user-images.githubusercontent.com/12775440/99282034-7bc6c780-2833-11eb-816a-b38de2827676.png)

Reply:
Height for the AppBar? Why do you need it? I was trying to respect spacing as much as possible. The AppBar should be rendered in a good shape without height specified.

Reply:
That's how our app structure works in search as well as in dashboard (not relevant). And the reasons I can think of only one at the moment and I already described it here https://github.com/oacore/design/pull/358#discussion_r524409693 but there were probably more reasons?

Reply:
@viktor-yakubiv Any update on this? What's your decision here? I see this as the last blocker for merging this PR.

Reply:
I finally got what's your problem is. Sorry, I did not get it before. This is how it looks for me:

<img width="795" alt="Screenshot 2020-11-23 at 11 15 54" src="https://user-images.githubusercontent.com/8440244/99945414-44996e80-2d7d-11eb-88a2-85613e5ecac5.png">

I will have a look at what I can do with `position: relative`

Reply:
Please, have a look at the [solution](https://github.com/oacore/design/pull/358/commits/326c2f95a255b7cb6d784aba5f1cfc7681a6cb3d). Doesn't it feel bug-causing?

Reply:
I hope it's ok. Please remove `FIXME` comment, because it will stay there forever. 😄 and after that feel free to merge. Thanks a lot for your work :)